scripts/template_matcher.py

```diff
@@ -7,7 +8,7 @@
 
 class TemplateMatcher(object):
 
-    def __init__ (self, images, min_match_count=10, good_thresh=0.7):
+    def __init__ (self, images, min_match_count=10, good_thresh=0.8):
```

Comment:
Might want to add a comment about what good_thresh is, and why you chose to make this default.  I see in one of your commits that increasing good thresh made better matches!  Document this in the code too.